resources/js/Shared/LoadingButton.vue

```diff
@@ -1,9 +1,13 @@
+<style lang="scss" scoped>
+@import 'vue-loaders/dist/vue-loaders.css';
+</style>
+
 <template>
   <div class="di">
     <button :class="classes" name="save" type="submit" :data-cy="cypressSelector" :disabled="state == 'loading'"
             @click="$emit('click')"
     >
-      <ball-pulse-loader v-if="state == 'loading'" color="#fff" size="7px" />
+      <ball-pulse-loader v-if="state == 'loading'" color="#fff" />
```

Comment:
@djaiss The `size` attribute doesn't do anything

Reply:
True